package.json

```diff
@@ -18,10 +19,17 @@
   "license": "MIT",
   "dependencies": {
     "@actions/core": "1.4.0",
-    "@actions/github": "5.0.0",
+    "@actions/github": "5.0.0"
+  },
+  "devDependencies": {
+    "@types/node": "^12.20.15",
     "@vercel/ncc": "0.28.6",
-    "@types/node": "12.20.15",
-    "typescript": "4.3.2",
-    "prettier": "2.3.1"
+    "husky": "^7.0.0",
```

Comment:
Could we go without the husky, and just stick to a more traditional `npm run lint` script that runs the styling on demand?